### PR TITLE
Increase quotas in appstudio and appstudiolarge

### PR DESCRIPTION
**Pull request by hugares**
appstudio:
* Parametrize request storage, keep same value of 200Gi
* Set build CPU request to half of limit, i.e. 60.

appstudiolarge:
* Set request storage to double of appstudio tier, i.e. 400Gi.
* Set build CPU request to half of limit, i.e. 240.
* Set build memory request to half of limit, i.e. 256Gi.

[KFLUXINFRA-632](https://issues.redhat.com//browse/KFLUXINFRA-632)

### CODE REVIEW

**Review by mfrancisc — APPROVED** [review]

**Comment by mfrancisc** [comment]
I doesn't look related to your changes : https://app.codecov.io/gh/codeready-toolchain/host-operator/pull/1045/indirect-changes

Even because you are not changing any go code in this PR. Feel free to merge , once you have the required approvals.

**Comment by hugares** [comment]
@mfrancisc Can you approve and merge, I do not have any rights here?

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**
